src/Core/src/WindowOverlay/WindowOverlay.Windows.cs

```diff
@@ -39,10 +40,11 @@ public virtual bool Initialize()
 				_frame.Navigating += FrameNavigating;
 			}
 
-			_graphicsView = new W2DGraphicsView() { Drawable = this };
-			if (_graphicsView == null)
+			if (!OperatingSystem.IsWindowsVersionAtLeast(10,0,18362))
```

Comment:
I will revert this and all other windows related updates after https://github.com/dotnet/Microsoft.Maui.Graphics/pull/382 merged and could be consumed by the repo

Reply:
This is reverted